schemas/ping.schema.json

```diff
@@ -0,0 +1,394 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Ping transport",
+  "description": "Schema for the ping content sent by Mozilla's glean telemetry SDK\n",
+  "$id": "http://mozilla.org/schemas/glean/ping",
```

Comment:
Should this use a URI that actually resolves to the schema?

Reply:
The JSON Schema standard says it doesn't have to, and rarely does.  However, that seems unnecessarily hostile ;)  If we were to have a real resolving URL, what would that be?

Reply:
That's the million dollar question - and I don't have a great answer.  What I was going to suggest is to use something that's clearly not supposed to resolve, like `moz://a/schemas/glean/lean.1.json`

Reply:
Not a bad idea.  Certainly easy enough.